### PR TITLE
chore(icons): update icons.yml

### PR DESCRIPTION
**Pull request by LMapes**
Updated icons metadata, normalized friendly names.

Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

{{ Add descriptions, steps or a checklist for how reviewers can verify this PR works or not }}

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]

File: packages/icons/icons.yml [hunk 1]
```diff
@@ -13987,15 +14696,20 @@
...
+    - warning
+    - forecast
   sizes:
     - 32
 - name: wintry-mix
   friendly_name: Wintry mix
   aliases:
     - wintry-mix
     - precipitation
+    - forecase
```
Comment (emyarod) [inline packages/icons/icons.yml]:
```suggestion
    - forecast
```

File: packages/icons/icons.yml [hunk 2]
```diff
@@ -7398,38 +7564,67 @@
...
+    - document
   sizes:
     - 32
 - name: license--maintenance-draft
-  friendly_name: license--maintenance-draft
-  aliases: []
+  friendly_name: License maintenance draft
+  aliases: 
+    - license
+    - maintenace
```
Comment (emyarod) [inline packages/icons/icons.yml]:
```suggestion
    - maintenance
```

**Review by joshblack — APPROVED** [review]

File: packages/icons/icons.yml [hunk 3]
```diff
@@ -13619,22 +14282,26 @@
     - inspect
     - visible
 - name: view-next
-  friendly_name: view-next
-  aliases: []
+  friendly_name: View next
+  aliases: 
+    - view next
```
Comment (andreancardona) [inline packages/icons/icons.yml:14304]: `- view`

should we add`view` as an alias?

File: packages/icons/icons.yml [hunk 4]
```diff
@@ -12886,33 +13482,59 @@
...
+    - incident
+    - transportation
   sizes:
     - 32
 - name: traffic-cone
-  friendly_name: traffic-cone
-  aliases: []
+  friendly_name: Traffic cone
+  aliases: 
+    - traffic flow incident
```
Comment (andreancardona) [inline packages/icons/icons.yml:13543]: `- traffic cone`
`- traffic`
`- cone`

File: packages/icons/icons.yml [hunk 5]
```diff
@@ -12340,23 +12896,51 @@
...
+    - cold
+    - min
   sizes:
     - 32
 - name: temperature--water
-  friendly_name: temperature--water
-  aliases: []
+  friendly_name: Temperature water
+  aliases: 
+    - weather
```
Comment (andreancardona) [inline packages/icons/icons.yml:12944]: `- temperature`

File: packages/icons/icons.yml [hunk 6]
```diff
@@ -12340,23 +12896,51 @@
...
+    - hot
+    - cold
+    - max
   sizes:
     - 32
 - name: temperature--min
-  friendly_name: temperature--min
-  aliases: []
+  friendly_name: Temperature min
+  aliases:
```
Comment (andreancardona) [inline packages/icons/icons.yml:12929]: `- temperature`

File: packages/icons/icons.yml [hunk 7]
```diff
@@ -12340,23 +12896,51 @@
...
+    - hot
+    - cold
+    - inversion
   sizes:
     - 32
 - name: temperature--max
-  friendly_name: temperature--max
-  aliases: []
+  friendly_name: Temperature max
+  aliases:
```
Comment (andreancardona) [inline packages/icons/icons.yml:12915]: `- temperature`

File: packages/icons/icons.yml [hunk 8]
```diff
@@ -12340,23 +12896,51 @@
     - hot
     - cold
 - name: temperature--inversion
-  friendly_name: temperature--inversion
-  aliases: []
+  friendly_name: Temperature inversion
+  aliases:
```
Comment (andreancardona) [inline packages/icons/icons.yml:12901]: `- temperature`

File: packages/icons/icons.yml [hunk 9]
```diff
@@ -12313,8 +12862,15 @@
   sizes:
     - 32
 - name: temperature--feels-like
-  friendly_name: temperature--feels-like
-  aliases: []
+  friendly_name: Temperature feels-like
```
Comment (andreancardona) [inline packages/icons/icons.yml:12866]: `- temperature`

File: packages/icons/icons.yml [hunk 10]
```diff
@@ -11699,33 +12221,54 @@
...
   sizes:
     - 32
 - name: soil-temperature--global
-  friendly_name: soil-temperature--global
-  aliases: []
+  friendly_name: Soil temperature global
+  aliases: 
+    - soil
+    - temperature
+    - agriculture
```
Comment (andreancardona) [inline packages/icons/icons.yml:12272]: `- global`

File: packages/icons/icons.yml [hunk 11]
```diff
@@ -7398,38 +7564,67 @@
...
+    - license
+    - third-party
+    - file
+    - document
   sizes:
     - 32
 - name: license--third-party-draft
-  friendly_name: license--third-party-draft
-  aliases: []
+  friendly_name: license third-party draft
```
Comment (andreancardona) [inline packages/icons/icons.yml]: `friendly_name: License third-party draft`

capital L

File: packages/icons/icons.yml [hunk 12]
```diff
@@ -5903,22 +6037,39 @@
     - 32
 - name: forecast--hail
   friendly_name: forecast--hail
```
Comment (andreancardona) [inline packages/icons/icons.yml]: @LMapes none of the forecasts look like friendly names can you please confirm: 

- [x]  `friendly_name: forecast--hail`
- [x]  `friendly_name: forecast--hail-30`
- [x] `friendly_name: forecast--lightning`
- [x]  ` friendly_name: forecast--lightning-30`
_- [ ] `friendly_name: forecast--lightning-30`_ accidental duplicate on. my end

- [x] `  friendly_name: ice--accretion`

File: packages/icons/icons.yml [hunk 13]
```diff
@@ -8549,7 +8888,9 @@
     - 32
 - name: monster
   friendly_name: monster
```
Comment (andreancardona) [inline packages/icons/icons.yml]: All friendly names seem to be capitalized - I'm assuming this on too? 

  `friendly_name: Monster`

**Review by andreancardona — CHANGES_REQUESTED** [review]

File: packages/icons/icons.yml (hunk 3 above)
Comment (tw15egan) [reply packages/icons/icons.yml:14304]: won't `view` match with `view next`?

File: packages/icons/icons.yml (hunk 13 above)
Comment (LMapes) [reply packages/icons/icons.yml]: Yes. Please initial cap all friendly names

File: packages/icons/icons.yml (hunk 3 above)
Comment (LMapes) [reply packages/icons/icons.yml:14304]: I would enter it as
 - view,
as well as leave 
- view next

File: packages/icons/icons.yml (hunk 4 above)
Comment (LMapes) [reply packages/icons/icons.yml:13543]: - travel
- transportation
- roadwork
- construction
as well as what you have andrea

File: packages/icons/icons.yml (hunk 5 above)
Comment (LMapes) [reply packages/icons/icons.yml:12944]: - forecast

File: packages/icons/icons.yml (hunk 6 above)
Comment (LMapes) [reply packages/icons/icons.yml:12929]: - forecast
as well Andrea

File: packages/icons/icons.yml (hunk 7 above)
Comment (LMapes) [reply packages/icons/icons.yml:12915]: - forecast
as well Andrea

File: packages/icons/icons.yml (hunk 9 above)
Comment (LMapes) [reply packages/icons/icons.yml:12866]: - forecast
as well Andrea
On all of these, I've tried to consistently mark aliases that apply to the context of the icon. Sorry I missed this forecast series

File: packages/icons/icons.yml (hunk 10 above)
Comment (LMapes) [reply packages/icons/icons.yml:12272]: fine by me

File: packages/icons/icons.yml (hunk 11 above)
Comment (LMapes) [reply packages/icons/icons.yml]: nice catch!

File: packages/icons/icons.yml (hunk 12 above)
Comment (LMapes) [reply packages/icons/icons.yml]: Forecast hail
Forecast hail 30
Forecast lightning
Forecast lightning 30
Forecast lightning 60 ??? Not sure about this, is a duplicate or is there a 60 as well?
Ice accretion

**Review by andreancardona — APPROVED** [review]

**Comment by andreancardona** [comment]
@LMapes looks good to merge to me!

**Pull request merged**